DEVELOPMENT.md

```diff
@@ -39,13 +39,25 @@
 
 ## Run locally
 
-You should be able to run `./scripts/localbuild.sh` and see your changes to the
-website and docs repo reflected. Note that the script **copies** the data from
-`../docs`, so you'll need to re-run the script to get fresh docs changes to show
-up on the local site.
+You should be able to run `./scripts/localbuild.sh` to generate a copy of the
+docs in the `public/` folder. Note that the build will replace relative
+`.../index.html` links with `.../`, so when browsing the local copy, you may
+need to click on `index.html` files to get where you need to go.
 
-> @evankanderson is working on
-> [fixing this](https://github.com/knative/website/issues/158)
+If you want the old behavior of starting a local webserver, you can run
```

Comment:
Due to the differences in the way the site is built and hosted online VS how the local webserver works, using only the local server to test the site can and did cause a lot of pain and trouble (early on during site development).

I think for simply testing if an authored markdown file "looks" right and all the links work, this sounds like an improvement for sure (will get around to testing next week). 

But I think since that this is the DEVELOPMENT.md file, we should be clear about how the local server can seem like everything works perfect but can then fails completely when deployed to Netlify.

Maybe:

Testing that authored markdown files render properly 
VS. 
Building the site how it gets hosted (ie. use this if new site features need local testing)